### PR TITLE
[Java Lab] Add an eyes test for the prompter

### PR DESCRIPTION
**Pull request by molly-moen**
Add an eyes test for the prompter feature of the theater in Java Lab. This also required adding a new allthethings level that starts with some prompter code.

## Links

- jira ticket: [CSA-1074](https://codedotorg.atlassian.net/browse/CSA-1074)

## Testing story
Tested locally.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/locales/scripts.en.yml [hunk 1]
```diff
@@ -1299,6 +1299,8 @@ en:
               name: AI Lab
             Java Lab:
               name: Java Lab
+              description_student: 
+              description_teacher:
```
Comment (sanchitmalhotra126) [inline dashboard/config/locales/scripts.en.yml:1303]: (for my understanding) what are these used for?

File: dashboard/config/scripts_json/allthethings.script_json [hunk 2]
```diff
@@ -3969,6 +3969,9 @@
       "activity_section_position": 4,
       "assessment": false,
       "properties": {
+        "level_keys": [
+          "gamelab free play"
+        ]
```
Comment (sanchitmalhotra126) [inline dashboard/config/scripts_json/allthethings.script_json:3974]: (also for my understanding) are these changes auto-generated, or something we need?

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice!

File: dashboard/config/locales/scripts.en.yml (hunk 1 above)
Comment (molly-moen) [reply dashboard/config/locales/scripts.en.yml:1303]: these were auto-generated, they match other scripts so I left them in

File: dashboard/config/scripts_json/allthethings.script_json (hunk 2 above)
Comment (molly-moen) [reply dashboard/config/scripts_json/allthethings.script_json:3974]: auto-generated

**Pull request merged**